### PR TITLE
Fix direct links in remote tests

### PR DESCRIPTION
**Pull request by rosteen**
The direct links for the files in mosviz_test_data got changed, updating them so the tests will stop failing. I also noted which files the links point to.

### CODE REVIEW

File: jdaviz/configs/mosviz/tests/test_parsers.py [hunk 1]
```diff
@@ -180,7 +181,7 @@ def test_nircam_parser(mosviz_helper, tmp_path):
     '''
 
     # Download data
-    test_data = 'https://stsci.box.com/shared/static/itk7pav073nubwn58pig002m9796qzpw.zip'
+    test_data = 'https://stsci.box.com/shared/static/g3tymg7c1yj67gy0hpfreo80inqdm1ps.zip'
```
Comment (pllim) [inline jdaviz/configs/mosviz/tests/test_parsers.py:184]: What is the actual file name for this one?

**Review by pllim — APPROVED** [review]
Thanks for tracking them down!

File: jdaviz/configs/mosviz/tests/test_parsers.py (hunk 1 above)
Comment (rosteen) [reply jdaviz/configs/mosviz/tests/test_parsers.py:184]: Whoops, good catch.

**Comment by pllim** [comment]
Ooops I can fix the milestone checker separately.

**Pull request merged**